src/52.py

```diff
@@ -0,0 +1,28 @@
+"""Создайте декоратор, который хранит результаты вызова функции
+(за все время вызовов, не только текущий запуск программы)
+"""
+
+
+def result_function(func):
+    def wrapper(*args, **kwargs):
+        res = func(*args, **kwargs)
+        with open('result.txt', 'a') as r:
+            r.write(res)

```

Comment:
it would be better to add name, datetime and result - we will have no advantages from this file without it